uint8 rACT #requestActivation
uint8 rGTO #requestActionStatus
uint8 rATR #requestAutomaticRelease
uint8 rPR #requestPosition
uint8 rSP #requestSpeed
uint8 rFR #requestForce
